src/studio/src/designer/frontend/shared/components/AltinnSpinner.tsx

```diff
@@ -6,7 +6,7 @@ import altinnTheme from '../theme/altinnStudioTheme';
 export interface IAltinnSpinnerComponentProvidedProps {
   id?: string;
   spinnerText?: any;
-  styleObj?: object;
+  styleObj?: string;
```

Comment:
`styleObj` is an odd name for a `string` prop 😅 I know it wasnt added now, but is it a quick fix?

Reply:
Extracted the type for argument for className which is used. Name then makes more sense.

Reply:
Thats even better :) Nice!